Index: C:/source/wackget/vars.mk
===================================================================
--- C:/source/wackget/vars.mk	(revision 1562)
+++ C:/source/wackget/vars.mk	(revision 1563)
@@ -1,10 +1,10 @@
 
-VERSION=1.2.2
+VERSION=1.2.4
 DEBUG=0
 CC=cl
 LINK=link
 DEFS=/DWIN32 /D_WINDOWS /DOEMRESOURCE /DDEBUG=$(DEBUG) /DVERSION=\"$(VERSION)\"
-CFLAGS=/nologo /W3
+CFLAGS=/nologo /W3 -I "C:\Program Files\Microsoft Platform SDK\Include"
 LDFLAGS=/nologo /machine:ix86 /libpath:$(OUT_DIR)
 
 !IF "$(DEBUG)" == "1"
Index: C:/source/wackget/WackGet/about.c
===================================================================
--- C:/source/wackget/WackGet/about.c	(revision 1562)
+++ C:/source/wackget/WackGet/about.c	(revision 1563)
@@ -34,7 +34,7 @@
 
   center_window(dlg_wnd, GetParent(dlg_wnd));
 
-  SetWindowText(GetDlgItem(dlg_wnd, IDC_VERSION), "WackGet version 1.2.2b2");
+  SetWindowText(GetDlgItem(dlg_wnd, IDC_VERSION), "WackGet version 1.2.4-Mammoth");
 
   info= MALLOC(sizeof(about_info_t));
   memset(info, 0, sizeof(*info));
Index: C:/source/wackget/WackGet/main.obj
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream
Index: C:/source/wackget/WackGet/pch.pch
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream
Index: C:/source/wackget/WackGet/list.c
===================================================================
--- C:/source/wackget/WackGet/list.c	(revision 1562)
+++ C:/source/wackget/WackGet/list.c	(revision 1563)
@@ -7,6 +7,7 @@
   "Status",
   "Progress",
   "Speed",
+  0
 };
 
 int get_column_index(const char *name)
@@ -300,8 +301,8 @@
 
   /* Set up wget command-line */
   ptr= buf;
-  ptr+= sprintf(ptr, "%s --tries=0 --waitretry=1 --ca-certificate=cacert.pem --passive --progress=wack:%d",
-                wget_pn, GetCurrentProcessId());
+  ptr+= sprintf(ptr, "--tries=0 --waitretry=1 --ca-certificate=cacert.pem --passive --progress=wack:%d",
+                GetCurrentProcessId());
   if(uinfo->download_dir &&
      uinfo->download_dir[0]!='\0') {
     ptr+= sprintf(ptr, " --directory-prefix=\"%s\"", uinfo->download_dir);
Index: C:/source/wackget/WackGet/main.h
===================================================================
--- C:/source/wackget/WackGet/main.h	(revision 1562)
+++ C:/source/wackget/WackGet/main.h	(revision 1563)
@@ -77,7 +77,7 @@
 
   int num_threads;
   int num_queued;
-  long total_speed;
+  __int64 total_speed;
 } update_info_t;
 
 /* Callbacks */
Index: C:/source/wackget/WGDLL/main.obj
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream
